ai-agent/api/services/markdown.py

```
@@ -0,0 +1,11 @@
+from mdutils.mdutils import MdUtils
```

Ruff (0.8.2)

`mdutils.mdutils.MdUtils` imported but unused

Remove unused import: `mdutils.mdutils.MdUtils`

(F401)